src/advanced_inv.cpp

```diff
@@ -1202,7 +824,7 @@ bool advanced_inventory::move_all_items( bool nested_call )
     if( spane.get_area() == AIM_ALL ) {
         // move all to `AIM_WORN' doesn't make sense (see `MAX_WORN_PER_TYPE')
         if( dpane.get_area() == AIM_WORN ) {
-            popup( _( "You look at the items, then your clothes, and scratch your head…" ) );
+            popup( _( "You look at the items, then your clothes, and scratch your head�" ) );
```

Comment:
You have invalid encoding here:

```suggestion
            popup( _( "You look at the items, then your clothes, and scratch your head…" ) );
```